packages/backend/src/common/logger/connectionMonitor.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { DataSource } from 'typeorm';
+
+@Injectable()
+export class ConnectionMonitorService {
+  constructor(private readonly datasource: DataSource) {}
+
+  getConnectionPoolInfo() {
+    const pool = (this.datasource.driver as any).pool;
+    if (!pool) {
+      return '';
+    }
```

Comment:
빈문자열보다는 상태를 명확히 표시하는건 어떤가요?
```suggestion
if (!pool) {
      return '[Connections: Not Available]';  // 명확한 상태 표시
    }
```

Reply:
해당 부분에 걸리게 되는 경우가 현재로서는 서버 시작 초기 단계만 존재해서 깊게 생각하지 못했습니다. 일단 말씀주신 것처럼 명확한 상태 표시를 함께하면 더 좋을 것 같네요.